### PR TITLE
Writing Day: Doc Detective and Docs as Test

### PR DESCRIPTION
**Pull request by rosewms**
First project for Writing Day!

----
📚 Documentation preview 📚: https://writethedocs-www--2100.org.readthedocs.build/

### CODE REVIEW

**Comment by rosewms** [comment]
I'm also sending this PR to the project organizer for review.

I often make edits to the survey content to ensure it makes sense for the Writing Day Project List - since style and formatting is different in surveys vs web pages.

File: docs/conf/portland/2024/writing-day.rst [hunk 1]
```diff
@@ -106,7 +106,24 @@ If you need additional information to advocate for Writing Day in your community
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Meet the team, get set up with tools or strategies, run some tests.**
+
+`Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.
+
+Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks on projects using automation. By validating that doc contents work as written, you can: 
+
+* Prevent broken docs.
+* Ensure consistent UX between docs, projects, or products.
```
Comment (hawkeyexl) [inline docs/conf/portland/2024/writing-day.rst]:
```suggestion
* Ensure consistent UX between docs and products.
```

File: docs/conf/portland/2024/writing-day.rst [hunk 2]
```diff
@@ -106,7 +106,24 @@ If you need additional information to advocate for Writing Day in your community
...
 
-Incoming project information! Check back here for updates closer to the conference.
+Docs as Tests & Doc Detective: Help us test your docs!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Meet the team, get set up with tools or strategies, run some tests.**
+
+`Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.
+
+Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks on projects using automation. By validating that doc contents work as written, you can:
```
Comment (hawkeyexl) [inline docs/conf/portland/2024/writing-day.rst]:
```suggestion
Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks directly in your products using automation. By validating that doc contents work as written, you can: 
```

File: docs/conf/portland/2024/writing-day.rst [hunk 3]
```diff
@@ -106,7 +106,24 @@ If you need additional information to advocate for Writing Day in your community
...
+
+Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks on projects using automation. By validating that doc contents work as written, you can: 
+
+* Prevent broken docs.
+* Ensure consistent UX between docs, projects, or products.
+* Build user trust.
+
+Our Writing Day goal is to test at least 10 docs projects! We want every writer to be confident in their docs. 
+
+Come by our table, and we'll help you set up Docs as Tests with whatever tools (such as Doc Detective) or strategies (like unit testing code snippets) are appropriate. Establish a zero-trust relationship between your docs and project, catching bugs in both as you go.
```
Comment (hawkeyexl) [inline docs/conf/portland/2024/writing-day.rst]:
```suggestion
Come by our table, and we'll help you set up Docs as Tests with whatever tools (such as Doc Detective) or strategies (like unit testing code snippets) are appropriate. Establish a zero-trust relationship between your docs and product, catching bugs in both as you go.
```

File: docs/conf/portland/2024/writing-day.rst [hunk 4]
```diff
@@ -106,7 +106,24 @@ If you need additional information to advocate for Writing Day in your community
...
+
+`Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.
+
+Docs as Tests is a tool-agnostic strategy for validating doc content by performing documented tasks on projects using automation. By validating that doc contents work as written, you can: 
+
+* Prevent broken docs.
+* Ensure consistent UX between docs, projects, or products.
+* Build user trust.
+
+Our Writing Day goal is to test at least 10 docs projects! We want every writer to be confident in their docs.
```
Comment (hawkeyexl) [inline docs/conf/portland/2024/writing-day.rst]:
```suggestion
Our Writing Day goal is to test at least 10 docs sets! We want every writer to be confident in their docs. 
```

**Review by hawkeyexl — CHANGES_REQUESTED** [review]
Clarified "project"/"product".

File: docs/conf/portland/2024/writing-day.rst [hunk 5]
```diff
@@ -106,7 +106,24 @@ If you need additional information to advocate for Writing Day in your community
 Project List
 ------------
 
-Incoming project information! Check back here for updates closer to the conference.
+Docs as Tests & Doc Detective: Help us test your docs!
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Meet the team, get set up with tools or strategies, run some tests.**
+
+`Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.
```
Comment (plaindocs) [inline docs/conf/portland/2024/writing-day.rst]:
```suggestion
`Doc Detective <https://doc-detective.com/>`__ is back at Writing Day again, this time with our friends, `Docs as Tests <https://www.docsastests.com>`__. Does your project or product have a UI? APIs? SDKs? We can help you test them and keep your docs accurate.
```

**Comment by rosewms** [comment]
Thanks @hawkeyexl and @plaindocs - Sam if you approve this, I can merge it.

**Review by plaindocs — APPROVED** [review]
:shipit:

**Pull request merged**